assignment2/day 4/assignment2.yaml

```diff
@@ -0,0 +1,777 @@
+swagger: '2.0'
+info:
+  description: >-
+    The Root resource provides information on all available resources within the
+    API.
+  version: 1.0.0
+  title: Root
+  contact:
+    email: vmishra@grepruby.com
+host: www.swapi.co
+basePath: /api
+tags:
+  - name: films
+    description: The URL root for Film resources
+    externalDocs:
+      description: Find out more
+      url: 'http://swapi.co/api/films/'
+  - name: people
+    description: The URL root for People resources
+    externalDocs:
+      description: Find out more
+      url: 'http://swapi.co/api/people/'
+  - name: planets
+    description: The URL root for Planet resources
+    externalDocs:
+      description: Find out more
+      url: 'http://swapi.co/api/planets/'
+  - name: species
+    description: The URL root for Species resources
+    externalDocs:
+      description: Find out more
+      url: 'http://swapi.co/api/species/'
+  - name: starships
+    description: The URL root for Starships resources
+    externalDocs:
+      description: Find out more
+      url: 'http://swapi.co/api/starships/'
+  - name: vehicles
+    description: The URL root for Vehicles resources
+    externalDocs:
+      description: Find out more
+      url: 'http://swapi.co/api/vehicles/'
+schemes:
+  - http
+paths:
+  /films/:
+    get:
+      tags:
+        - Films
+      summary: get all the film resources
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Films'
+  '/films/{:id}/':
+    get:
+      tags:
+        - Films
+      summary: get a specific film resource
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: ':id'
+          in: path
+          description: id of the film to be returned
+          required: true
+          type: integer
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Films'
+  /films/schema/:
+    get:
+      tags:
+        - Films
+      summary: view the JSON schema for this resource
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Films'
```

Comment:
Hit the _/films/schema/_ once and see if it returns you this response or something else.